### PR TITLE
{Release} Upgrade to Azure CLI 2.63.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,100 @@
...
+++++++
+
+**ACR**
+
+* `az acr login`: Support optional resource group argument to skip subscription resource scan (#29344)
+
+**AKS**
+
+* `az aks create`: Not set the `--network-plugin` based on the default from the Python SDK (#29388)
+* `az aks create/update`: Support in place param updates for managed prom (#29273)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:16]:
```suggestion
* `az aks create/update`: Support in place param updates for managed Prometheus (#29273)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,100 @@
...
+* `az role assignment list`: Add warning for classic administrators retirement (#29404)
+
+**Service Connector**
+
+* `az containerapp connection create containerapp`: Support ACA2ACA connection (#29434)
+
+**SQL**
+
+* `az sql midb move/copy`: Add destination subscription Id for managed database move/copy (#29352)
+* `az sql mi create`: Add `--dns-zone-partner` optional parameter (#29381)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:89]:
```suggestion
* Fix #27422: `az sql midb move/copy`: Add destination subscription Id for managed database move/copy (#29352)
* `az sql mi create`: Add `--dns-zone-partner` optional parameter (#29381)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,100 @@
...
+* `az aks create`: Remove unsupported scenario command in help message (#29387)
+
+**App Config**
+
+* `az appconfig import/export/restore`: Add correlation request id to bulk operations (#29252)
+
+**App Service**
+
+* `az webapp list-runtimes`: Add parameter `--show-runtime-details` to show detailed runtime stacks and update the format of java related stacks listed (#29367)
+* `az webapp create`: Add parameter `--acr-identity` to allow users choose user assigned identity they want to use for ACR image pull (#29321)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:29]:
```suggestion
* `az webapp create`: Add parameter `--acr-identity` to allow users choose user assigned identity for ACR image pull (#29321)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,100 @@
...
+
+**Network**
+
+* `az network custom-ip prefix create`: Add parameter `--is-parent` (#29350)
+* `az network network-watcher connection-monitor`: Support to create connection monitor v2 (#29288)
+* `az network vnet peering`: Support virtual network subnet peering (#29416)
+
+**RDBMS**
+
+* [BREAKING CHANGE] Update default value of PG version to be 16 for `postgres flexible-server create` (#29443)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:74]:
```suggestion
* [BREAKING CHANGE] `az postgres flexible-server create`: Update default value of PG version to be 16 (#29443)
```

File: src/azure-cli-core/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+2.63.0
+++++++
+* Resolve CVE-2024-39689 (#29320)
+* Set `allow-preview` from config for extension dynamic installation (#29413)
```
Comment (AllyW) [inline src/azure-cli-core/HISTORY.rst:9]:
```suggestion
* Add `extension.dynamic_install_allow_preview` for extension dynamic installation configuration (#29413)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,100 @@
...
+
+**Container app**
+
+* Fix #26688: `az containerapp up`: Fix logic about updating an existing containerapp (#29336)
+* `az containerapp job stop`: Return custom message for stop job execution (#29399)
+* Fix #29330: `az containerapp auth update`: Fix split logic for `--set` (#29453)
+
+**Cosmos DB**
+
+* `az cosmosdb delete`: Add no-wait (#29190)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:56]:
```suggestion
* `az cosmosdb delete`: Support `--no-wait` (#29190)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,100 @@
...
+
+* `az network custom-ip prefix create`: Add parameter `--is-parent` (#29350)
+* `az network network-watcher connection-monitor`: Support to create connection monitor v2 (#29288)
+* `az network vnet peering`: Support virtual network subnet peering (#29416)
+
+**RDBMS**
+
+* [BREAKING CHANGE] Update default value of PG version to be 16 for `postgres flexible-server create` (#29443)
+* `az postgres flexible-server create`: Bug fix for using existing subnet while creating pg flex server (#29457)
+* `az postgres flexible-server restore`: Bug fix when using resource id as value for source-server argument (#29460)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:76]:
```suggestion
* [BREAKING CHANGE] Update default value of PG version to be 16 for `postgres flexible-server create` (#29443)
* * Fix #27422: `az postgres flexible-server create`: Bug fix for using existing subnet while creating pg flex server (#29457)
* `az postgres flexible-server restore`: Bug fix when using resource id as value for source-server argument (#29460)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,100 @@
...
+**Network**
+
+* `az network custom-ip prefix create`: Add parameter `--is-parent` (#29350)
+* `az network network-watcher connection-monitor`: Support to create connection monitor v2 (#29288)
+* `az network vnet peering`: Support virtual network subnet peering (#29416)
+
+**RDBMS**
+
+* [BREAKING CHANGE] `az postgres flexible-server create`: Update default value of PG version to be 16 (#29443)
+* `az postgres flexible-server create`: Bug fix for using existing subnet while creating pg flex server (#29457)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:75]:
```suggestion
* Fix #27422: `az postgres flexible-server create`: Bug fix for using existing subnet while creating pg flex server (#29457)
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Pull request merged**